config/pickit.ini

```diff
@@ -177,9 +176,9 @@ magic_unearthed_wand=0
 
 ;magic assassin claws
 magic_greater_claws=0
-magic_greater_talons=0
+magic_greater_talons=1, and(2_assasin_skills, 3_lightning_sentry)
 magic_feral_claws=0
-magic_runic_talons=0
+magic_runic_talons=1, and(2_assasin_skills, 3_lightning_sentry)
```

Comment:
So we dont allow +3 trap, +3 lightning? Maybe only filter for +3_lightning?

Reply:
changed to pickup 2 assa, 3 Lightning and traps+ lightning sentry.
((2_assasin_skills, 3_lightning_sentry), (traps, lightning_sentry))